tuple-merger-scm-1.rockspec

```diff
@@ -1,20 +1,20 @@
-package = 'mergerx'
+package = 'tuple-merger'
 version = 'scm-1'
 
 -- url and branch of the package's repository at GitHub
 source  = {
-    url    = 'git://github.com/tsafin/tarantool-merge.git';
+    url    = 'git://github.com/tarantool/tuple-merger.git';
     branch = 'tarantool-merge';
```

Comment:
Should be 'master' in the foreseeable future?

Reply:
Thanks for the catch! Yes, indeed!